src/Infrastructure/JobSchedulers/NomadJobScheduler.cs

```diff
@@ -201,6 +201,10 @@ private string JobDefinition(Channel c)
       tags = [
         ""traefik.enable=true"",
         ""traefik.http.routers." + name + @".rule=Host(`${var.host}`)"",
+        ""traefik.http.routers." + name + @".entryPoints=websecure"",
+        ""traefik.http.routers." + name + @".tls=true"",
+        ""traefik.http.routers." + name + @".tls.certresolver=letsencrypt"",
```

Comment:
These two additions imply/assume:
- there is expected to be an entryPoint named `websecure` and
- there is expected to be a certresolver named `letsencrypt` configured on the Traefik instance that will be serving this app, which may not be the case.

We should see how the job deploys (or not) if one or both assumptions are faulty.

Reply:
Jobs still run even if both the entryPoint or certresolver do not exist... Or even if Traefik does not exist for that matter.